src/main/java/ch/wisv/areafiftylan/security/authentication/AuthenticationServiceImpl.java

```diff
@@ -74,10 +74,6 @@ public void removeAuthToken(String xAuth) {
         AuthenticationToken token =
                 authenticationTokenRepository.findByToken(xAuth).orElseThrow(XAuthTokenNotFoundException::new);
 
-        if (!token.isValid()) {
```

Comment:
Why is this removed?

Reply:
This just sets a boolean. The thrown exception here causes a bunch of pointless stacktraces in the log. Worst case, you set the boolean to false twice. Best case, no more stacktraces.

Reply:
If the token is already not valid, why not just `return` here? Saves a write to the database and maybe sideeffects from calling `revoke()` again.

Reply:
There could be different reasons for an invalid token. 

`isUsed() || isExpired() || isRevoked()`

An explicit revoke seems like a good idea here, even if we don't have a direct use now.